rclcpp/minimal_service/main.cpp

```diff
@@ -18,23 +18,26 @@
 #include "rclcpp/rclcpp.hpp"
 
 using AddTwoInts = example_interfaces::srv::AddTwoInts;
+rclcpp::Node::SharedPtr g_node = nullptr;
 
 void handle_service(
   const std::shared_ptr<rmw_request_id_t> request_header,
   const std::shared_ptr<AddTwoInts::Request> request,
   const std::shared_ptr<AddTwoInts::Response> response)
 {
   (void)request_header;
-  printf("request: %" PRId64 " + %" PRId64 "\n", request->a, request->b);
+  RCLCPP_INFO(
+    g_node->get_logger(),
```

Comment:
Segfaults since the global variable is never set.

Reply:
Actually this example works fine. I think you're referring to the not composable subscriber. See #195 for a fix